main.c

```diff
@@ -147,8 +147,6 @@ int main(void)
 
     atecc608a_print_serial_number();
     atecc608a_print_config_zone();
-    ASSERT_SUCCESS_PSA(atecc608a_generate_key(atecc608a_key_slot_device, pubkey, pubkey_size));
-    atcab_printbin_label("pubKey generated: ", pubkey, ATCA_PUB_KEY_SIZE);
```

Comment:
Do we need to update https://github.com/ARMmbed/mbed-os-example-atecc608a/blob/master/tests/atecc608a.log, since the prints are changing?

Reply:
No, since we're not checking for this log anyway, and all the tests are in series. The logs and the logfile itself will change here: https://github.com/ARMmbed/mbed-os-example-atecc608a/pull/7